src/cycles/johnson.jl

```diff
@@ -265,14 +270,14 @@ after a given number of cycles.
 - [Johnson](http://epubs.siam.org/doi/abs/10.1137/0204007)
 """
 function itercycles end
-@traitfn function itercycles(dg::::IsDirected, cycle::Channel)
+@traitfn function itercycles(dg::AG::IsDirected, cycle::Channel{Vector{T}}) where {T, AG <: AbstractGraph{T}}

```

Comment:
Our informal convention is to use `where T where AG <: AbstractGraph{T}` instead.

Reply:
I just tested that, apparently `@traitfn` does not support multiple `where` clauses.
```
ERROR: LoadError: LoadError: LoadError: Could not parse function-head: (itercycles((dg::AG)::IsDirected, cycle::Channel{Vector{T}}) where T) where AG <: AbstractGraph{T}. Note that several `where` are not supported.
```

Reply:
Ah, right. Sorry.